app/views/admin/content_pages/_details_for_a_single_page.erb

```diff
@@ -9,7 +9,7 @@
           |<%= link_to 'Add a child page', new_admin_content_page_path(parent_id: page_to_render.id), { class: "govuk-link" } %>
         <% end %>
         <% if (page_to_render.is_published) %>
-          | <%= link_to 'Preview', path_for_this_page(page_to_render), { class: "govuk-link", target: "_blank" } %>
+          | <%= link_to 'Preview', preview_of_live_admin_content_page_path(@content_page, id: page_to_render.id), { class: "govuk-link", target: "_blank" } %>
```

Comment:
That path looks wrong. It should just take the id (or an object that has the id) of a content page. So either:
```ruby
preview_of_live_admin_content_page_path(@content_page)
```
or
```ruby
preview_of_live_admin_content_page_path(page_to_render)
```
But not both.